clusterloader2/testing/load/config.yaml

```diff
@@ -50,6 +50,9 @@
 # Probe measurements shared parameter
 {{$PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT := DefaultParam .CL2_PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT "5m"}}
 
+# Command to be executed
+{{$EXEC_COMMAND := DefaultParam .EXEC_COMMAND nil}}
```

Comment:
Let's use the `.CL2_EXEC_COMMAND` to make it configurable via env var

Reply:
It's array, it cannot be modified by ENV anyway

Reply:
But will rename